server/controllers/user_controller.js

```diff
@@ -30,17 +30,14 @@ class UserController {
         return res.status(status.REQUEST_CONFLICT).json({
           status: status.REQUEST_CONFLICT,
           error: `${email} already exists`,
-
         });
       }
       password = await encryptPassword(password);
-
       const columns = 'first_name, last_name, email, password, address, bio, occupation, expertise, is_mentor, is_admin';
       const data = `'${first_name}', '${last_name}', '${email}', '${password}','${address}','${bio}','${occupation}','${expertise}',${is_mentor},${is_admin}`;
       const rows = await this.model().insert(columns, data) || [];
       if (rows.length) {
         let token = generateAuthToken(rows[0].id, rows[0].is_mentor, rows[0].is_admin);
```

Comment:
You defined the token within this method but you're not using it.

Reply:
it is going to be returned when user signed up

Reply:
Okay.